Feature branch vincent

aduuna opened the pull request:
fixes #103 and #86

aduuna committed:
set flag for which folder to deploy

aduuna committed:
Updated gitigonore for generated dagster files

aduuna committed:
dagster config yaml

aduuna committed:
added independent student dagster home

oliverangelil commented on .github/workflows/ build_and_deploy.yml:
```diff
@@ -60,7 +60,7 @@ env:
   REPOSITORY: '${{ secrets.GCP_PROJECT_NAME }}-gcr'
   WORKLOAD_IDENTITY_PROVIDER: 'projects/${{ secrets.PROJECT_NUMBER }}/locations/global/workloadIdentityPools/github/providers/github'
-  DEPLOY_EXAMPLE: 'true' # <--- Toggle this to 'true' to deploy the example folder
+  DEPLOY_EXAMPLE: 'false' # <--- Toggle this to 'true' to deploy the example folder
```
I can't find where this variable is used? You define it but don't use it?

aduuna replied:
I didn't want my merge to trigger a deployment  that overrides the example folder, just in case you or @cyrille-feu have any recent deployment to the root gcp

oliverangelil replied:
Ok let's discuss in the upcoming call with @cyrille-feu before we merge

aduuna committed:
changed the toggle for example folder

aduuna committed:
added dynamic student config for dagster.yaml

aduuna committed:
completed configurations for dagster data write into student directory

aduuna committed:
fixed file write issue of non existent folder write

aduuna committed:
updated dagster run configurations with workspace.yaml config

aduuna committed:
first test run student - aduuna

aduuna committed:
removed all user specific data

aduuna commented:
In this PR,

changed ML pipeline code to write raw data files to `$DAGSTER_HOME/data/`

Also modified how `dagster dev` is started -> Now uses `run_dagster.sh`, which automatically configures the environment and creates all neccesary directories

oliverangelil committed:
.
